docs/source/poly.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+from datetime import datetime
+from functools import partial
+from sphinx_polyversion import *
+from sphinx_polyversion.git import *
+from sphinx_polyversion.git import closest_tag
+from sphinx_polyversion.pyvenv import Poetry
+from docs.source.builder import DffSphinxBuilder
+
+#: Regex matching the branches to build docs for
+# BRANCH_REGEX = r"((?!master).)*"
+# Put all branches here except master, so docs can be built for any branch
+# if the workflow is launched from it.
+BRANCH_REGEX = r".*"
+
+
+#: Regex matching the tags to build docs for
+TAG_REGEX = r"(v0.8.0)"
+# That was just 0.8.0, should change that to auto tags at some point.
+# Could just check always what version is being released, but I'm not sure how to do that yet.
+# For now this means someone has to change this on every single release to whatever the release tag will be. Potentially, if a mistake is made, this can be changed in dev and launched from there, just don't forget to fetch 'master' branch.
+
+#: Output dir relative to project root
+OUTPUT_DIR = "docs/build"
+
+#: Source directory
+SOURCE_DIR = "docs/source"
+
+#: Arguments to pass to `poetry install`
+POETRY_ARGS = "--with tutorials,docs --all-extras --no-ansi --no-interaction".split()
+
+#: Arguments to pass to `sphinx-build`
+SPHINX_ARGS = "-b html -W --keep-going -v".split()
+
+#: Mock data used for building local version
+MOCK_DATA = {
+    "revisions": [
+        GitRef("dev", "", "", GitRefType.BRANCH, datetime.fromtimestamp(0)),
+        GitRef("master", "", "", GitRefType.BRANCH, datetime.fromtimestamp(1)),
+    ],
+    "current": GitRef("local", "", "", GitRefType.BRANCH, datetime.fromtimestamp(2)),
+}
+MOCK = False
```

Comment:
Local building is now done and configured for local multiversioning. Developers can run `poetry run poe docs` just like before and it will work. It only builds for one git branch at a time. (the current branch)

Reply:
> BTW, AFAIR you said something about polyversion taking pre-built docs from `gh-pages`. Could you remind me what that was about?

I don't recall saying that. I think I could be bad at explaining and was just thinking of how to save docs on gh-pages branch. I don't think sphinx-polyversion itself can do things like that.